### PR TITLE
feat(driver): add macbook-16 viewport

### LINKED ISSUE #8889: add viewport for macbook 16'
### Current behavior
Currently, we don't have a preset of MacBook 16'

### Desired behavior
Need to add a preset of MacBook 16' in our codebase.

### Test code to reproduce

```
beforeEach(function () {
    cy.viewport('macbook-15')
    cy.login();
});

describe('Home page', () => {
    it('Home Page', () => {
        cy.navigateTo('Home');  
       
        cy.wait(2000);
    });
});

```
If you see in beforeEach for cy.viewport(''), we don;t have macbook-16' available.
Screenshot 2020-10-20 at 12 09 16 PM

### Versions

Cypress: 5.4.0, MacOS 10.15.7 (19H2), Google Chrome:86.0.4240.80

### PR DESCRIPTION
**Pull request by sagarrajput**
* Closes #8889

### User facing changelog
* Added viewport preset of macbook-16

### Additional details
 improve Developer Experience
viewport preset consider useful most people because people tend to memorize devices by naming e.g. macbook-15, macbook-16 etc clearly more than device pixels number.

### How has the user experience changed?
Most of the Mac users are using MacBook-16 and I don't see its viewport preset available in cypress, so now it will be easy to find that viewport.

### PR Tasks

* [x] Have tests been added/updated?
* [ ] Has the original issue been tagged with a release in ZenHub? 
* [x] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? https://github.com/cypress-io/cypress-documentation/pull/3260
* [x] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
* [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: cli/types/cypress.d.ts
Comment (jennifer-shehane) [inline cli/types/cypress.d.ts:5298]:
```suggestion
  type ViewportPreset = 'macbook-16' | 'macbook-15' | 'macbook-13' | 'macbook-11' | 'ipad-2' | 'ipad-mini' | 'iphone-xr' | 'iphone-x' | 'iphone-6+' | 'iphone-se2' | 'iphone-8' | 'iphone-7' | 'iphone-6' | 'iphone-5' | 'iphone-4' | 'iphone-3' | 'samsung-s10' | 'samsung-note9'
```

**Review by jennifer-shehane — DISMISSED** [review]

**Review by jennifer-shehane — DISMISSED** [review]

**Review by jennifer-shehane — APPROVED** [review]

**Comment by sagarrajput** [comment]
@jennifer-shehane Can this be merged?

Thanks

**Comment by jennifer-shehane** [comment]
@sagarrajput Please do not merge in the `develop` branch. We need to get all the tests to pass which only requires re-running a few of the failing tests. Merging in the `develop` branch starts this all over from scratch - running all the tests, delaying this from being merged in. As soon as all the tests pass we can merge this in.

**Comment by sagarrajput** [comment]
@jennifer-shehane oops, I will wait till all tests pass.

**Pull request merged**
